documentation/source/Policies/storage_policy.rst

```diff
@@ -0,0 +1,127 @@
+
+Storage Policies
+==================
+You can now define policies based on the type of application at the storage-class level. Following are some of the properties that can be customized at the default level in the openebs-storageclasses.yaml file.
+
+* jiva-replica-count
```

Comment:
Let us mention along with `openebs.io/`
i.e. openebs.io/jiva-replica-count
this is applicable for all policies.

Reply:
Done!